### ISSUE TITLE
Warning: include_once(): Failed opening...

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/views_accordion#6]
It doesn't seem to affect functionality but I am seeing this in the log:

> Warning: include_once(): Failed opening '/app/modules/views_accordion/templates/views_accordion.views.inc' for inclusion (include_path='.:/usr/local/lib/php') in include_once() (line 1059 of /app/core/includes/theme.inc).

Adding a placeholder file in this default location solves this warning.

**Comment by laryn** [comment]
Closed with https://github.com/backdrop-contrib/views_accordion/commit/7d927e16e13903cd511cb4ae8e6bc6c9f2158492

**Comment by herbdool** [comment]
I got this error too. I saw your fix (but not in a release yet). I tried to see why it was giving this error here but presumably in core/modules/views/tests/views_test, which has a similar setup, they're not getting the same error. I wasn't able to eliminate the error. I tried moving `views_accordion_views_plugins()` to .module and deleting views_accordion.views.inc. Even though the module still worked the error message was too. Odd.
